app/models/item.rb

```diff
@@ -4,11 +4,16 @@ class Item < ApplicationRecord
   has_many :invoices, through: :invoice_items
   has_many :customers, through: :invoices
 
+  enum status: {
+    disabled: 0,
+    enabled: 1
+  }
+
   def quantity(invoice)
     InvoiceItem.where(item_id: id, invoice_id: invoice).first.quantity
   end
 
-  def status(invoice)
+  def invoice_status(invoice)
```

Comment:
Great idea on the name change to support the status enum you created for the `Items` model.